packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -72,7 +73,44 @@ export function makeWallet({
   myAddressNameAdmin,
   pursesStateChangeHandler = noActionStateChangeHandler,
   inboxStateChangeHandler = noActionStateChangeHandler,
+  localTimerService,
 }) {
+  /**
+   * The current timestamp, in milliseconds (if it is known).
+   *
+   * @type {number | undefined}
+   */
+  let nowMs;
+
+  // Subscribe to the timer service to update our stamp.
+  if (localTimerService) {
+    observeNotifier(E(localTimerService).makeNotifier(0n, 1000n), {
+      updateState(bigStamp) {
+        // We need to convert the bigint to a number (for easy Javascript date math).
+        nowMs = parseInt(`${bigStamp}`, 10);
+      },
+    });
+  }
+
+  let lastSequence = 0;
+  const addMeta = record => {
```

Comment:
Nit: Maybe a doc comment here would be helpful to see what metadata it adds, maybe even an example json.

Reply:
Sure, I'll add a comment to this effect:

```js
record.meta = { sequence: number, creationStamp: number, updatedStamp: number };
```

Where the Stamps are in milliseconds.